storage/scylla_table.c

```diff
@@ -0,0 +1,110 @@
+/*
+ * Copyright (C) 2019 BiiLabs Co., Ltd. and Contributors
+ * All Rights Reserved.
+ * This is free software; you can redistribute it and/or modify it under the
+ * terms of the MIT license. A copy of the license can be found in the file
+ * "LICENSE" at the root of this distribution.
+ */
+#include "scylla_table.h"
+#include "common/model/transaction.h"
+#include "utils/logger.h"
+
+static logger_id_t logger_id;
+
+void scylla_table_logger_init() { logger_id = logger_helper_enable("scylla_table", LOGGER_DEBUG, true); }
+
+int scylla_table_logger_release() {
+  logger_helper_release(logger_id);
+  if (logger_helper_destroy() != RC_OK) {
+    log_critical(logger_id, "%s.\n", "scylla_table");
+    return EXIT_FAILURE;
+  }
+  return 0;
+}
+
+struct identity_table_s {
```

Comment:
Do we need `static` modifier here?

Reply:
Struct identity_table_s is a definition, not a declaration.
reference: https://stackoverflow.com/questions/2743949/useless-class-storage-specifier-in-empty-declaration?rq=1